### ISSUE TITLE
Default GKE version failing

### ISSUE THREAD

**Issue by ocervell** [terraform-google-modules/terraform-google-kubernetes-engine#26]
The default version for GKE master (1.10.6-gke.2) has been [deprecated](https://cloud.google.com/kubernetes-engine/release-notes#october-22-2018).

Currently running with the default version throws:

```
* google_container_cluster.primary: googleapi: Error 400: Master version "1.10.6-gke.2" is unsupported., badRequest
```

**Comment by morgante** [comment]
I think #24 should fix this.

**Comment by ocervell** [comment]
Ok, i'll keep it opened until the PR is merged.

**Comment by morgante** [comment]
Fixed by #24
